packages/calcite-components/src/components/block/block.scss

```diff
@@ -3,16 +3,23 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-block-padding: Specifies the padding of the block `default` slot.
+ * @prop --calcite-block-border-color: Specifies the border color of the component.
+ * @prop --calcite-block-header-background-color: Specifies the background color of the component's header.
+ * @prop --calcite-block-header-background-color-hover: Specifies the background color of the component's header when hovered.
+ * @prop --calcite-block-header-text-color: Specifies the text color of the component's header.
+ * @prop --calcite-block-padding: [Deprecated] Specifies the padding of the component's `default` slot.
+ * @prop --calcite-block-text-color: Specifies the text color of the component.
+ * @prop --calcite-block-text-color-hover: Specifies the text color component when hovered
  */
 
 :host {
   @extend %component-host;
   @extend %component-spacing;
-  @apply transition-margin ease-cubic border-color-3 flex flex-shrink-0 flex-grow-0
+  @apply transition-margin ease-cubic flex flex-shrink-0 flex-grow-0
     flex-col border-0 border-b border-solid p-0;
   flex-basis: auto;
   transition-duration: var(--calcite-animation-timing);
+  border-color: var(--calcite-block-border-color, var(--calcite-color-border-3));
```

Comment:
add here
`color: var(--calcite-block-text-color, var(--calcite-color-text-3))`
setting it here should mean we can reduce lines of code by removing the need to set `color` further down the component style cascade.
Double check styles still look correct

Reply:
After adding it, I tested removing it in other spots further down the component and it didn't really work.